packages/react-components/react-charts-preview/library/src/components/CommonComponents/CartesianChart.types.ts

```diff
@@ -425,6 +425,12 @@ export interface ICartesianChartProps {
    * @param {number} height - The new height of the chart.
    */
   onResize?: (width: number, height: number) => void;
+
+  /**
+   * Define a custom callout props override
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  customProps?: (dataPointCalloutProps: any) => IPopoverComponentProps;
```

Comment:
give this a better name. Why disabling eslint rule in this line

Reply:
Lets take the comment in next PR. Merging the changes to unblock PR to master.

Reply:
this is because different charts can have different types. For example, VBC has type IVerticalBarChartDataPoint while Line chart has type ICustomizedCalloutData.